api/v1/service/resource/resource.go

```diff
@@ -508,13 +511,58 @@ func versionInfoFromVersion(v model.ResourceVersion) *resource.ResourceVersion {
 func (s *service) GetRawYamlByCatalogKindNameVersion(ctx context.Context, p *resource.GetRawYamlByCatalogKindNameVersionPayload) (io.ReadCloser, error) {
 	s.Logger(ctx).Info(fmt.Sprintf("Fetching YAML for resource %s", p.Name))
 
+	if p.Version == Latest {
+		version, err := getLatestVersion(s.DB(ctx), p.Name, p.Kind)
+		if err != nil {
+			return nil, err
+		}
+		p.Version = version
+	}
+
```

Comment:
I think there is no need to to modify this api and add latest in the api endpoint as we are already adding one more api to print the latest raw yaml

Reply:
cc @piyush-garg @vinamra28